invisible_cities/detector_simulation/buffer_functions_test.py

```diff
@@ -0,0 +1,109 @@
+import numpy  as np
+import pandas as pd
+
+from pytest import mark
+
+from .. core            import system_of_units as units
+
+from . buffer_functions import       bin_sensors
+from . buffer_functions import buffer_calculator
+from . buffer_functions import find_signal_start
+
+
+def test_bin_sensors(mc_waveforms, pmt_ids, sipm_ids):
+    max_buffer = 10 * units.minute
+
+    evts, pmt_binwid, sipm_binwid, all_wfs = mc_waveforms
+
+    evt   = evts[0]
+    wfs   = all_wfs.loc[evt]
+
+    pmts  = wfs[wfs.index.isin( pmt_ids)]
+    sipms = wfs[wfs.index.isin(sipm_ids)]
+
+    ## Assumes pmts the triggering sensors as in new/next-100
+    bins_min = pmts.time.min()
+    bins_max = pmts.time.max() + pmt_binwid
+    pmt_bins ,  pmt_wf = bin_sensors(pmts ,  pmt_binwid,
+                                     bins_min, bins_max, max_buffer)
+    sipm_bins, sipm_wf = bin_sensors(sipms, sipm_binwid,
+                                     bins_min, bins_max, max_buffer)
+
+    assert  pmt_bins[ 0] >= bins_min
+    assert  pmt_bins[-1] <= min(bins_max, max_buffer)
+    assert np.all(np.diff( pmt_bins) ==  pmt_binwid)
+    assert sipm_bins[ 0] >= bins_min
+    assert sipm_bins[-1] <= min(bins_max, max_buffer)
+    assert np.all(np.diff(sipm_bins) == sipm_binwid)
+    ## In current DAQ, pmts have higher sample frequency than SiPMs
+    assert pmt_bins[ 0] >= sipm_bins[ 0]
+    assert pmt_bins[-1] >= sipm_bins[-1]
```

Comment:
```suggestion
    assert  pmt_bins[ 0] >= bins_min
    assert  pmt_bins[-1] <= min(bins_max, max_buffer)
    assert np.all(np.diff( pmt_bins) ==  pmt_binwid)ç
    
    assert sipm_bins[ 0] >= bins_min
    assert sipm_bins[-1] <= min(bins_max, max_buffer)
    assert np.all(np.diff(sipm_bins) == sipm_binwid)
    
    ## In current DAQ, pmts have higher sample frequency than SiPMs
    assert pmt_bins[ 0] >= sipm_bins[ 0]
    assert pmt_bins[-1] >= sipm_bins[-1]
```
just some spaces for readability